docs/src/main/sphinx/release/release-470.md

```diff
@@ -0,0 +1,102 @@
+# Release 470 (5 Feb 2025)
+
+## General
+
+* Prevent failures of the `ARRAY_HISTOGRAM` function resulting from null values. ({issue}`24765`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+* {{breaking}} Raise minimum runtime requirement to Java 11. ({issue}`23639`)
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+* {{breaking}} Raise minimum runtime requirement to Java 11. ({issue}`23639`)
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+## Druid connector
+
+## Elasticsearch connector
+
+## Exasol connector
+
+## Faker connector
+
+## Google Sheets connector
+
+## Hive connector
+
+## Hudi connector
+
+## Iceberg connector
+
+* Fix failure when adding a new column with a name containing a dot. ({issue}`24813`)
+
+## Ignite connector
+
+## JMX connector
+
+## Kafka connector
+
+## Kinesis connector
+
+* {{breaking}} Remove the Kinesis connector. ({issue}`23923`) 
+
+## Kudu connector
+
+## MariaDB connector
+
+## Memory connector
+
+## MongoDB connector
+
+## MySQL connector
+
+* Add support for `MERGE` statement. ({issue}`24428`)
+* Fix incorrect writing of negative date values to the database. ({issue}`24809`)
```

Comment:
fyi @wendigo

Reply:
So "Prevent writing of invalid negative date values" ?

Reply:
yes